graphics/src/ColladaExporter.cc

```diff
@@ -41,24 +41,10 @@ static void LogTinyXml2DocumentError(
   {
     warning += std::to_string(_doc.ErrorID()) + "): ";
 
-#ifdef TINYXML2_MAJOR_VERSION_GE_6
```

Comment:
Have you checked the TinyXML2 versions on all our supported platforms?

Reply:
I reverted this.